### PR TITLE
Canvas API Get/Post Documentation

### PR DESCRIPTION
**Pull request by jSunpayco**
This PR is for the issue below
[Get and Post API Requests to Canvas](https://github.com/code4sac/learning-blocks/issues/31)

### CODE REVIEW

**Comment by jSunpayco** [comment]
@natebass You mentioned something about documenting Python syntax. I already have some of it in the first part of this markdown. Would you like me to move it to another file or make this one more detailed?

**Comment by natebass** [comment]
@jSunpayco I the Python related stuff you put in that file is good where it's at.

File: Documentation Directory/Cavas_api_documentation.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+
+This is where the specific endpoints must be concatenated at.
+
+## Authentication
+
+Ensure that an API Token has been obtained before sending requests, and define them as such: `headers = {'Authorization': 'Bearer [api_token_here]'}`
+
+## Sending requests
+
+Using the requests dependency to fetch data as a JSON object: `requests.get(url, headers=headers)`
```
Comment (natebass) [inline Documentation Directory/Cavas_api_documentation.md:13]: You could capitalize the name Requests dependency. That's how they reference themselves [Requests website](https://docs.python-requests.org/en/latest/index.html)

**Review by natebass — APPROVED** [review]

**Comment by jSunpayco** [comment]
@natebass Just pushed the change for spelling the Requests dependency. Thanks!

**Pull request merged**